src/olympia/lib/settings_base.py

```diff
@@ -779,6 +779,7 @@ def get_db_config(environ_var, atomic_requests=True):
     'olympia.users.tasks.resize_photo': {'queue': 'amo'},
     'olympia.users.tasks.update_user_ratings_task': {'queue': 'amo'},
     'olympia.versions.tasks.delete_preview_files': {'queue': 'amo'},
+    'olympia.hero.tasks.sync_primary_hero_addon': {'queue': 'amo'},
```

Comment:
You can use the `adhoc` queue for one-off tasks like these - it has dedicated workers. See a few lines above this block